lib/datadog/profiling/exporter.rb

```diff
@@ -65,6 +69,7 @@ def flush
           code_provenance_data: uncompressed_code_provenance,
           tags_as_array: Datadog::Profiling::TagBuilder.call(settings: Datadog.configuration).to_a,
           internal_metadata: internal_metadata,
+          info: info,
```

Comment:
I wonder if we should cache the `internal_metadata` and the `info` as JSON in the constructor, since right now we repeat the work of encoding it every time.